cheddar/cheddar-integration-aws/src/main/java/com/clicktravel/infrastructure/messaging/aws/sqs/DefaultSqsQueueResourceFactory.java

```diff
@@ -56,7 +56,6 @@ public SqsQueueResource createSqsQueueResource(final String name, final SnsTopic
         final String queueUrl = amazonSqsClient.getQueueUrl(new GetQueueUrlRequest(name)).getQueueUrl();
         logger.info("Using existing SQS queue: " + name);
         final SqsQueueResource sqsQueueResource = new SqsQueueResource(name, queueUrl, amazonSqsClient);
-        subscribeToSnsTopics(name, sqsQueueResource, snsTopics);
```

Comment:
- With the removal of this line, `snsTopics` is no longer required in this method (`createSqsQueueResource`) and should be removed.
- Also note, removing this line may cause issues for existing applications that add subscriptions to event topics in application revisions after the (subscribing) event queue is first created i.e. the subscriptions to the extra SNS topics will not be added. What is the harm in leaving this line for now and removing it after we have developed our other changes in provisioning?